definitions/divorce/json/CaseField.json

```diff
@@ -3210,5 +3202,15 @@
     "Label": "Decree Absolute Granted Date",
     "FieldType": "Date",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "GrantDA",
```

Comment:
I think this should be in the past tense. DAGranted, for example.
I was a bit confusing to me to understand what this meant.

Reply:
Agreed, thanks